packages/forma-36-codemod/utils/changeImportWithComponentName.js

```diff
@@ -0,0 +1,18 @@
+const { addImport } = require('./addImport');
+const { removeComponentImport } = require('./removeComponentImport');
+
+module.exports.changeImportWithComponentName = function changeImport(
```

Comment:
can we just modify `changeImport` function to accept optional outputComponentName property?

Reply:
Sure. Dunno, thought it would be useful to have a separate function for some reason 🙂. I'm gonna extend the `changeImport`.

Reply:
✅